pkg/infrastructure/gcp/clusterapi/network.go

```diff
@@ -31,48 +32,71 @@ func getInternalLBAddress(ctx context.Context, project, region, name string) (st
 	return addrOutput.Address, nil
 }
 
-// createInternalLBAddress creates a static ip address for the internal load balancer.
-func createInternalLBAddress(ctx context.Context, in clusterapi.InfraReadyInput) (string, error) {
+// createInternalLB creates a static ip address for the internal load balancer.
+// Returns the IP address of the created load balancer.
+func createInternalLB(ctx context.Context, in clusterapi.InfraReadyInput, subnetSelfLink, networkSelfLink string, zones []*string) (string, error) {
+	projectID := in.InstallConfig.Config.GCP.ProjectID
+	region := in.InstallConfig.Config.GCP.Region
 	name := getAPIAddressName(in.InfraID)
 
-	// TODO: Find the self link for a subnet from the in.Client
-	// TODO: can we pick one returned from the service.Get() ?
-	subnetSelfLink := ""
+	service, err := NewComputeService()
+	if err != nil {
+		return "", err
+	}
+
+	ctx, cancel := context.WithTimeout(ctx, time.Minute*3)
+	defer cancel()
+
+	logrus.Debug("Patching external load balancer")
+	extBesvcName := fmt.Sprintf("%s-apiserver", in.InfraID)
+	extBesvc, err := service.BackendServices.Get(projectID, extBesvcName).Context(ctx).Do()
+	if err != nil {
+		return "", fmt.Errorf("failed to get backend service: %w", err)
+	}
+
+	for _, be := range extBesvc.Backends {
+		be.BalancingMode = "CONNECTION"
+		be.MaxConnections = int64(2 ^ 32)
+	}
 
-	// TODO: the subnet is only relevant for internal load balancer
+	op, err := service.BackendServices.Patch(projectID, extBesvcName, extBesvc).Context(ctx).Do()
```

Comment:
I know this code will probably go away when we create the Load Balancers in CAPG but it would be useful to have a comment here until then as to why this patch is necessary.